user-service/controller/user-controller.js

```diff
@@ -24,3 +26,20 @@ export async function createUser(req, res) {
     }
 }
 
+export async function signIn(req, res) {
+    try {
+        const { username, password } = req.body;
+        const user = await _FindOneuser(username,password)
+        if (user) {
+            const accessToken = jwt.sign({username:username},ACCESS_TOKENT_SECRET)

```

Comment:
Shouldn't the jwt access token be randomly generated instead of being a constant?

Reply:
For now we will keep it to be a constant, will be diving deeper into the use of jwt